docs/source/en/using-diffusers/inference_with_tcd_lora.md

```diff
@@ -0,0 +1,435 @@
+TCD-LoRAs are available for [stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5), [stable-diffusion-2-1-base](https://huggingface.co/stabilityai/stable-diffusion-2-1-base), and [stable-diffusion-xl-base-1.0](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0). 
+
+The corresponding checkpoints can be found at [TCD-SD15](https://huggingface.co/h1t/TCD-SD15-LoRA), [TCD-SD21-base](https://huggingface.co/h1t/TCD-SD21-base-LoRA), and [TCD-SDXL](https://huggingface.co/h1t/TCD-SDXL-LoRA), respectively.
+
+
+This guide shows how to perform inference with TCD-LoRAs for 
+- text-to-image
+- inpainting
+- community models
+- style LoRA
+- ControlNet
+- IP-Adapter
+- AnimateDiff
+
+TCD-LoRA can be considered an advanced method compared with [LCM-LoRA](https://latent-consistency-models.github.io/). The main parts of the TCD-LoRA workflow are as follows::
+- Load the task specific pipeline and model.
+- Set the scheduler to [`TCDScheduler`].
+- Load the TCD-LoRA weights for the model.
+- Set the `num_inference_steps` between [4, 50].
+- Set `eta` from [0, 1]. Larger `eta` in [`TCDScheduler`] will lead to blurrier images.
+- Perform inference with the pipeline with the usual parameters.
+
+Let's look at how we can perform inference with TCD-LoRAs for different tasks.
+
+First, make sure you have [peft](https://github.com/huggingface/peft) installed, for better LoRA support.
+
+```bash
+pip install -U peft
+```
+
+## Text-to-image
+
+You can use the [`StableDiffusionXLPipeline`] with the scheduler: [`TCDScheduler`] and then load the TCD-LoRA. Together with the TCD-LoRA and the TCDScheduler, the pipeline enables a fast inference workflow with high quality outputs.
+
+```python
+import torch
+from diffusers import StableDiffusionXLPipeline, TCDScheduler
+
+device = "cuda"
+base_model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+tcd_lora_id = "h1t/TCD-SDXL-LoRA"
+
+pipe = StableDiffusionXLPipeline.from_pretrained(base_model_id, torch_dtype=torch.float16, variant="fp16").to(device)
+pipe.scheduler = TCDScheduler.from_config(pipe.scheduler.config)
+
+pipe.load_lora_weights(tcd_lora_id)
+pipe.fuse_lora()
+
+prompt = "Painting of the orange cat Otto von Garfield, Count of Bismarck-Schönhausen, Duke of Lauenburg, Minister-President of Prussia. Depicted wearing a Prussian Pickelhaube and eating his favorite meal - lasagna."
+
+image = pipe(
+    prompt=prompt,
+    num_inference_steps=4,
+    guidance_scale=0,
+    eta=0.3, 
+    generator=torch.Generator(device=device).manual_seed(0),
+).images[0]
+```
+
+![](https://github.com/jabir-zheng/TCD/raw/main/assets/demo_image.png)
+
+
+<Tip>
+
+Eta (referred to as `gamma` in the paper) is used to control the stochasticity in every step.
+A value of 0.3 often yields good results, where eta = 0 means determinstic and eta = 1 is identity to Multi-step Consistency Sampler (as well as LCMScheduler).
+We recommend using a higher eta when increasing the number of inference steps.
+
+</Tip>
```

Comment:
Better to have the tips upfront where users get the full picture before running the code

```suggestion
```